packages/domonda-ui/src/Menu/Menu/Menu.tsx

```diff
@@ -4,30 +4,50 @@
  *
  */
 
-import * as React from 'react';
-import clsx from 'clsx';
+import React from 'react';
+import { Popover, PopoverProps, PopoverOrigin } from '../../Popover';
 
 // decorate
 import { decorate, Decorate } from './decorate';
 
-export interface MenuProps extends React.HTMLAttributes<HTMLElement> {
+type ExtendingProps = Omit<PopoverProps, 'classes'>;
+export interface MenuProps extends ExtendingProps {
   classes?: Decorate['classes'];
-  component?: string | React.ComponentType<React.HTMLAttributes<HTMLElement>>;
+  PopoverClasses?: PopoverProps['classes'];
 }
 
-const Menu = React.forwardRef<HTMLElement, MenuProps & Decorate>(function Menu(props, ref) {
+const Menu = React.forwardRef<HTMLDivElement, MenuProps & Decorate>(function Menu(props, ref) {
   const {
-    component: Component = 'nav' as React.ElementType<React.ComponentPropsWithRef<'nav'>>,
     children,
     classes,
-    className,
+    anchorOrigin = { vertical: 'bottom', horizontal: 'center' } as PopoverOrigin,
+    PopoverClasses,
+    PaperProps = {},
+    transformOrigin = {
+      vertical: 'top',
+      horizontal: 'center',
+    } as PopoverOrigin,
     ...rest
   } = props;
 
   return (
-    <Component {...rest} ref={ref} className={clsx(classes.root, className)}>
+    <Popover
+      ref={ref}
+      anchorOrigin={anchorOrigin}
+      classes={PopoverClasses}
+      PaperProps={{
+        ...PaperProps,
+        classes: {
+          root: classes.root,
+          rounded: '',
+          ...PaperProps.classes,
```

Comment:
`PaperProps.classes` shouldn't be overriding the `root` class. If you wan't to change the `root` class, you would pass a different class to the root components (`Menu`) classes.